dolphin/processor/config.py

```diff
@@ -81,7 +81,23 @@ def pixel_size(self):
         :return:
         :rtype:
         """
-        return self.settings['pixel_size']
+        if type(self.settings['pixel_size']) == float:
+            return [self.settings['pixel_size']]
```

Comment:
Can you make a check here where it sees how many bands there are, and then make the list size that long. In that way, the user can still provide just one float value for the pixel size that is same for multiple bands provided.